ci_framework/roles/install_yamls/README.md

```diff
@@ -1,5 +1,5 @@
 # install_yamls
-An ansible role wrapper around [install_yamls](https://github.com/openstack-k8s-operators/install_yamls) Makefile.
+An ansible role wrapper around [install_yamls](https://github.com/openstack-k8s-operators/install_yamls) Makefile. It dynamically creates `install_yamls_makes` role, which can be reused within CI Framework.
```

Comment:
if you like here is a pointer to where this is consumed as an example

https://github.com/rdo-infra/rdo-jobs/blob/39d0647cbb20abe3aaf2baad134a0e09473e1c54/playbooks/data_plane_adoption/ci_framework_install_yamls.yaml#L5-L24


"which can be used within the CI framework (e.g. link) "